common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -188,14 +189,14 @@ class Backend(
             "currency" to receiptInfo.currency,
             "attributes" to subscriberAttributes.takeUnless { it.isEmpty() },
             "normal_duration" to receiptInfo.duration,
-            "intro_duration" to receiptInfo.introDuration,
```

Comment:
we still have to figure out how to pass these values in amazon...

amazon offers work the way google used to -- they are automatically applied. so i don't think we can make PurchaseOptions with PricingPhases for those. 

either way, i'm not sure we have that info available to us. Product.toStoreProduct for amazon just passed null introPrice/freeTrial fields. @vegaro can you confirm this?

Reply:
Yeah, that's tricky. I think we can return a single purchase option with a single princing phase for Amazon? Although it's not really a pricing phase I think it makes for a consistent API

Reply:
Hmm this feels worse to me... Having null values could indicate that we don't have that information, which is correct for amazon. But if we have a single pricing phase for a product with a free trial, that's actually having incorrect information... 

Having said that, if there is no way to detect free trials in amazon and considering this is internal, I think it would be ok. We just need to be aware that pricing phases can contain incomplete information for Amazon. 

Thinking about possible alternatives, we could have different bodies or endpoints in our backend for different stores sending the information we need in a format that matches what we have for each store, but this would be a big change and I don't think it's needed right now so I'm ok with this for now.